scripts/globals/settings.lua

```diff
@@ -135,6 +135,9 @@ EXPLORER_MOOGLE_LV = 10 -- Enables Explorer Moogle teleports and sets required l
 HALLOWEEN_2005 = 0 -- Set to 1 to Enable the 2005 version of Harvest Festival, will start on Oct. 20 and end Nov. 1.
 HALLOWEEN_YEAR_ROUND = 0 -- Set to 1 to have Harvest Festival initialize outside of normal times.
 
+-- TRUSTS
+ENABLE_TRUST_QUESTS = 1
```

Comment:
Can we put this up higher near the other content/char based settings (caskets, books, outpost warps) ?

Maybe right beneath outposts before the shop section.